cs3api4lab/utils/model_utils.py

```diff
@@ -189,12 +190,12 @@ def create_empty_file_model(path):
         return model
 
     @staticmethod
-    def update_file_model(model, stat=None):
+    def update_file_model(model, mount_dir, stat=None):
```

Comment:
Nie rozumiem czemu po co rozszerzamy parametry funkcji tylko po to zeby zmodyfikowac nizej sciezke ktora tez jest podana wyzej? jak dla mnie ta zmiana jest bledna

Reply:
Rozumiem, że intencja jest taka, że skoro mount_dir jest znany (ustalony przy starcie systemu, z konfiguracji) to chcemy, żeby funkcja FileUtils.remove_mount_dir wiedziała jaki jest mount_dir (ergo nie potrzebowała tego parametru), więc parametr ten będzie też zbędny w funkcji update_file_model()